### ISSUE TITLE
Uncaught TypeError: The "data" argument must be one of type string, Buffer, TypedArray, or DataVi...

### ISSUE THREAD

**Issue by ryanegt** [dsone/remote-ssh#9]
[Enter steps to reproduce:]

1. Installed package from within atom
2. Used existing .ftpconfig that works with the remote-ftp package
3. CTRL+ALT+R

**Atom**: 1.47.0 x64
**Electron**: 5.0.13
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [Remote-SSH](https://github.com/dsone/remote-ssh) package 0.11.0

### Stack Trace

Uncaught TypeError: The "data" argument must be one of type string, Buffer, TypedArray, or DataView. Received type undefined

```
At internal/crypto/hash.js:64

TypeError [ERR_INVALID_ARG_TYPE] [ERR_INVALID_ARG_TYPE]: The "data" argument must be one of type string, Buffer, TypedArray, or DataView. Received type undefined
    at Hash.update (internal/crypto/hash.js:64:11)
    at Clipboard.md5 (~/AppData/Local/atom/app-1.47.0/resources/app/static/<embedded>:1:775513)
    at Clipboard.write (~/AppData/Local/atom/app-1.47.0/resources/app/static/<embedded>:1:775586)
    at /packages/Remote-SSH/lib/remote-ssh.js:259:25
    at FSReqCallback.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:54:3)
```

### Commands

```
     -0:46.1.0 remote-ssh:start (ul.list-inline.tab-bar.inset-panel)
```

### Non-Core Packages

```
atom-autocomplete-php 0.25.6 
busy-signal 2.0.1 
git-log 0.4.1 
git-time-machine 2.1.0 
ide-php 0.7.18 
intentions 1.1.5 
language-babel 2.85.0 
linter 2.3.1 
linter-php 1.6.1 
linter-ui-default 1.8.1 
php-composer-completion 0.3.4 
php-hyperclick 1.2.1 
php-integrator-autocomplete-plus 1.6.2 
remote-ftp 2.2.4 
Remote-SSH 0.11.0 
tree-view-git-status 1.5.3 
windows-carriage-return-remover 0.1.3 
```

**Comment by dsone** [comment]
Is it possible that you have "copy passwords" enabled but no `password` key inside your .ftpconfig?

**Comment by ryanegt** [comment]
This happens when trying to do pub/priv key authentication. The private key path is set. When disabling "copy password" the client opens a putty window to try authentication, but throws a PuTTY Fatal Error: Disconnected: No supported authentication methods available (server sent: publickey)

**Comment by dsone** [comment]
Should be fixed in v0.13, if not feel free to reopen.
